pkg/k8s/apis/cilium.io/v2/ccnp_types.go

```diff
@@ -91,14 +92,14 @@ func (r *CiliumClusterwideNetworkPolicy) Parse() (api.Rules, error) {
 
 	retRules := api.Rules{}
 
-	if r.Spec != nil {
+	switch {
```

Comment:
This switch is changing behavior, with this switch a policy with both spec and specs defiled wouldn't return any errors. We should keep this functionality but move the `if r.Spec == nil && r.Specs == nil` to line 94.

Reply:
Good point, will change.